### ISSUE TITLE
Wakes up but doesn't show the host is on-line

### ISSUE THREAD

**Issue by hughgilhespie** [basildane/WakeOnLAN#108]
Hi, yet another 1803 update problem. After updating both local and host machines I have the odd problem that WOL DOES wake up the host machine but never changes to show that the host machine is on-line. I am afraid that I know almost nothing about networks so I can't give much more information. 
I did try pinging the remote machine but this fails - "Ping request could not find host 00-E0-4C-68-0A-7A. Please check the name and try again.

When I look in Network in Windows Explorer, the remote computer does show up there and I can connect using RDP - which is what I need to do. It would be great if I could go back to using WOL to first wake up the remote computer then make an RDP connection.

Thanks, Hugh

**Comment by basildane** [comment]
OK, you need to use the machine's IP address in ping.
For example: ping 192.168.0.44

Please try again and report back.

**Comment by AJBello1437** [comment]
That's how I ping my pcs/devices. I use the ip address as I have my router static assign the address.
I not only use WOL to wake a Windows 10 PC by schedule (so acronis runs a full backup) but to quickly see online status (for devices like security cameras).
Since the latest WOL update the app has been very sluggish in responding where quite often I see not responding message in Windows.
My latest reinstall attempt was from a total clean install where I uninstalled wol, deleted all data folders it created, restarted Windows PC, reinstalled WOL, then had it go scan for devices on my network. Once it finds all the devices, and I add them, this is where it starts to hang.
Great app, just been a little buggy since last update. Is there a newer version planned?


Sent via the Samsung Galaxy Note8, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by AJBello1437** [comment]
Will do, but it's not just from 1803, I had responding issues from 1709 too.



Sent via the Samsung Galaxy Note8, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by basildane** [comment]
do any machines stay in the "unknown" category, or do they all end up in Online / Offline?

**Comment by AJBello1437** [comment]
Yes, about half come up with showing green, then the rest are gray but at that point, the app stops responding so I'm not sure if it because the app stopped reporting or it gets hung up on trying to verify a particular device.




Sent via the Samsung Galaxy Note8, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by basildane** [comment]
can you tell by looking at the screen which device it's having a problem with?  the tracelog should give a hint to that.

**Comment by AJBello1437** [comment]
That's what I thought at first too, that it's a particular device, but it varies each time I restart the app. And in trying to manipulate the app to set it not to ping, it's too late and starts unresponding again.



Sent via the Samsung Galaxy Note8, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by AJBello1437** [comment]
File has been uploaded.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by basildane** [comment]
AJBello1437, I've seen problems with people using an IP address for a machine name.  I'm still trying to duplicate the problem, but this could be related.

**Comment by AJBello1437** [comment]
Should I try renaming the devices that show an IP address?



Sent via the Samsung Galaxy Note8, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by basildane** [comment]
Yes, I know it's a bit of work.  I would do this, make a backup of your "machines.xml" file.  Then start with just one or a few machines and see what the problem is.

**Comment by AJBello1437** [comment]
Didn’t work. I renamed all the devices on the list that were named with an IP address, then turned on pinger and it locked on 1st one.

**Comment by basildane** [comment]
Wow.  Could you eliminate all the rest of the machines, leave just this one, then try it again?
And if it hangs, could you upload the machines.xml file to me?

**Comment by AJBello1437** [comment]
I managed to get to stop hanging now, but still performing very sluggishly. I’ll send you the machines.xml file shortly.
